hazardous/data/__init__.py

```diff
@@ -1,3 +1,3 @@
-from ._competing_weibull import make_synthetic_competing_weibull
+# from ._competing_weibull import make_synthetic_competing_weibull
 
-__all__ = ["make_synthetic_competing_weibull"]
```

Comment:
Please keep the existing `make_synthetic_competing_weibull` function unchanged and add yours as an additional function to this module.

The original `make_synthetic_competing_weibull` is still used by existing tests the other example, so we should not modify it.